test/format-test.cc

```diff
@@ -1906,9 +1906,9 @@ TEST(FormatTest, FormatterNotSpecialized) {
 }
 
 #if FMT_HAS_FEATURE(cxx_strong_enums)
-enum TestFixedEnum : short { B };
+enum TestFixedEnum : short { B = 1 };
```

Comment:
Is it actually formatted differently if we don't map to underlying type?

Reply:
I don't understand the problem clearly but if map() returns short (which doesn't enumerated in enum `type`) 
`
FMT_ENABLE_IF(std::is_enum<T>::value &&
                          !has_formatter<T, Context>::value &&
                          !has_fallback_formatter<T, Context>::value)>
  FMT_CONSTEXPR auto map(const T& val) -> decltype(static_cast<typename std::underlying_type<T>::type>(val)) {
    return static_cast<typename std::underlying_type<T>::type>(val);
  }
`
then after that will be called map(short) with correct argument and map(int) with garbage ( =0 almost always). It means that fmt::format(short_enum_value) will return "0" regardless of the argument.